tests/queries/0_stateless/02354_hnsw_index.reference

```diff
@@ -0,0 +1,143 @@
+--- Negative tests ---
+--- Test default GRANULARITY (should be 100 mio. for usearch)---
+CREATE TABLE default.tab\n(\n    `id` Int32,\n    `vector` Array(Float32),\n    INDEX usearch_index vector TYPE usearch GRANULARITY 1\n)\nENGINE = MergeTree\nORDER BY id\nSETTINGS index_granularity = 8192
```

Comment:
In l. 3 and l. 4, you find "GRANULARITY 1". That's wrong, it needs to be 1 million - the documentation of Annoy (the file you edited as well) gives the exact reasoning why the default granularity is different for ANN indexes in ClickHouse.

Please
- see the logic in Parsers/ParserCreateQuery.h/cpp and Parsers/ASTIndexDeclaration.h (grep for "annoy")
- add similar logic for hnsw, while at it it makes sense to rename the constant to `DEFAULT_ANN_INDEX_GRANULARITY`
- re-generate the expected results of the test